packages/components/Chip/src/Chip.tsx

```diff
@@ -0,0 +1,85 @@
+/** @jsxRuntime classic */
+/** @jsx withSlots */
+import type { ReactNode } from 'react';
+import { Children, useCallback } from 'react';
+import { Pressable, I18nManager } from 'react-native';
+
+import type { UseSlots } from '@fluentui-react-native/framework';
+import { compose, withSlots, mergeProps } from '@fluentui-react-native/framework';
+import { Icon, createIconProps } from '@fluentui-react-native/icon';
+import type { InteractionEvent } from '@fluentui-react-native/interactive-hooks';
+import { TextV1 as Text } from '@fluentui-react-native/text';
+
+import { stylingSettings } from './Chip.styling';
+import type { ChipType, ChipProps, ChipState } from './Chip.types';
+import { chipName } from './Chip.types';
+import { useChip } from './useChip';
+
+export const chipLookup = (layer: string, userProps: ChipProps, state: ChipState): boolean => {
```

Comment:
Should this component be Android only? Right now it's exposed to all platforms.

Reply:
Removing for all other platforms for now